core/metadata-3.2.md

```diff
@@ -120,6 +132,23 @@ using `ERR_KEYNOPERMISSION` with an asterisk (`*`) in the `<Key>` field.
 
 *Errors*: `ERR_KEYNOPERMISSION`
 
+### METADATA SYNC
+
+This subcommand requests the full synchronization of metadata associated with
```

Comment:
Nevermind, I misread the examples and reached the wrong conclusions. Also there's no guarantee that PRIORITY as described in the other ticket will be what is merged with this one. Maybe just delete these three comments.